src/AcmServiceProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\acm;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceModifierInterface;
+use Drupal\Core\DependencyInjection\ServiceProviderBase;
+
+/**
+ * Class AcmServiceProvider.
+ */
+class AcmServiceProvider extends ServiceProviderBase implements ServiceModifierInterface {
```

Comment:
I am confused about this and #6504

Reply:
class name needs to match module name

Reply:
it was more about potential duplication, we will need to discuss

Reply:
@sdelbosc we discussed, this is fine.